model/configuration.py

```diff
@@ -294,7 +291,6 @@ class ExternalIntegration(Base, HasFullTableCache):
     settings = relationship(
         "ConfigurationSetting",
         backref="external_integration",
-        lazy="joined",
```

Comment:
This one liner was causing a lot of chaos in any queries involving `ExternalIntegration` because it caused it to always do a join to ConfigurationSettings and return all the settings for an ExternalIntegration. Adding a join to the query and more data to process. But we never use these settings. They are accessed through `setting()` which doesn't use anything returned in `settings`.